astropy/modeling/tests/test_parameters.py

```diff
@@ -543,11 +542,6 @@ def test_shape(self):
         param = Parameter(name="test", default=np.array([1]))
         assert param.shape == (1,)
         # Reshape error
-        if not NUMPY_LT_1_25:
-            # error message changed in numpy 1.25
-            MESSAGE = r"cannot reshape array of size 1"
-        else:
-            MESSAGE = r"Cannot assign this shape to a scalar quantity"
```

Comment:
I think we need to keep this `MESSAGE` or it is going to be undefined.

I am also going to close/reopen to make basebranch check happy.

Reply:
I pushed a commit to fix this. FYI.